Makefile

```diff
@@ -54,6 +54,17 @@ push-tag: # example usage: make push-tag tag=v1.100.1 remote=origin
 	 	git push $(remote) "$${dir:2}/$(tag)"); \
 	done
 
+.PHONY: publish-godoc
+publish-godoc: # example usage: make publish-godoc tag=v1.100.1
+	@[ "$(tag)" ] || ( echo ">> 'tag' is not set"; exit 1 )
+	@echo "godoc for root"
+	@cd ; go get -u github.com/signalfx/signalfx-go-tracing@$(tag)
+	@set -e; cd; \
```

Comment:
What are the `cd` commands doing here?

Reply:
Changing the directory to home so that the `go.mod` is NOT modified

Reply:
Do we need to do this twice? Also, to ensure things in the home directory are not modified, should we change to a newly created temp directory?

Reply:
No. We can do it once 👍 

Never seen anyone to have `go.mod` in `$HOME`. The `cd && go get` pattern is pretty common. See: https://maelvls.dev/go111module-everywhere/ . As far as I remember [VS Code Go extension](https://github.com/golang/vscode-go) is using it to install the Go tools.

Reply:
> The cd && go get pattern is pretty common.

Understood. Doing this in a temp directory will avoid the issue entirely and is a pattern for us to start setting as a good example for others. :smile: 

You should be able to support both Linux and BSD variants with:

```sh
tmpdir=$(mktemp -d 2>/dev/null || mktemp -d -t 'goget')
```

Reply:
PTAL https://github.com/signalfx/signalfx-go-tracing/pull/159/commits/50582022f7ba163d2d9a47d147d2ea12b26c2f75